Python/PRP/Animation/pyKeys.h

```diff
@@ -17,17 +17,48 @@
 #ifndef _PYKEYS_H
 #define _PYKEYS_H
 
+#include <PRP/Animation/hsKeys.h>
```

Comment:
Why not use forward declarations?

Reply:
That wouldn't allow me to call `key->getType()` in the switch statement